### ISSUE TITLE
ContentProvider Generator does not support order by clause

### ISSUE THREAD

**Issue by R4md4c** [Raizlabs/DBFlow#404]
Hello,

I'm using DBFlow's ContentProvider generator and the provider is generated correctly, but the problem is that the order clause is not added to the Select statement in the provider.
Here is an example

```Java
@Override
  public final Cursor query(Uri uri, String[] projection, String selection, String[] selectionArgs, String sortOrder) {
    Cursor cursor = null;
    switch(MATCHER.match(uri)) {
      case Company_CONTENT_URI: {
        cursor = new Select(projection)
            .from(FlowManager.getTableClassForName("CompanyDatabase", "Company"))
            .where(selection, selectionArgs)
            .query();
        break;
      }
```
As you can see from the above code the sortOrder string is not used and if used from a cursor loader results aren't returned as expected.

I think this should be the correct generated code
```Java
@Override
  public final Cursor query(Uri uri, String[] projection, String selection, String[] selectionArgs, String sortOrder) {
    Cursor cursor = null;
    switch(MATCHER.match(uri)) {
      case Company_CONTENT_URI: {
        cursor = new Select(projection)
            .from(FlowManager.getTableClassForName("CompanyDatabase", "Company"))
            .where(selection, selectionArgs)
            .orderBy(sortOrder) // Notice the order by clause addition.
            .query();
        break;
      }
```

**Comment by R4md4c** [comment]
I've submitted a pull request, let me know what do you think about it.

**Comment by ivanonchi** [comment]
I bumped into this issue as well. Any chances to have @R4md4c PR merged?

**Comment by agrosner** [comment]
fixed in 3.0.0-beta4
